clients/python/test/e2e/ci/setup_e2e_pytest.py

```diff
@@ -0,0 +1,121 @@
+import toml
+from pathlib import Path
+import pandas as pd
+from urllib.parse import urlparse, ParseResult
+from typing import List, Dict, Any, Union
+import typer
+from packaging import version
+import json
+from _utils import E2eScriptFailure, E2eExitCodes
+import warnings
+
+# keys to be found in input dicts
+surl: str = "OSPARC_API_HOST"
```

Comment:
STYLE: constants should be capitalized
e.g.
```python
SURL: Final[str] = "OSPARC_API_HOST"
```
Check https://github.com/ITISFoundation/osparc-simcore/blob/bf838aa1a559f060428536b17ad05e5876129982/docs/coding-conventions.md#python

Reply:
fixed